### DISCUSSION TITLE
Curious why the usage of google-translate-api-x was reverted

### DISCUSSION THREAD

**Discussion by AlexXi19** [SillyTavern/SillyTavern#3062] [General]
![img_v3_02gh_d8fe9fb7-f547-429a-b938-3365eae2101g](https://github.com/user-attachments/assets/3d560520-e53b-4277-991f-3a4b626c42f8)
@Cohee1207

**Comment by Cohee1207** [comment]
> i have a huge issue with the update, now after chat translation my chat is weird, and it doesnt know how to use ** and "" anymore, i even resetted al my setting to stock standard and it still doesnt work. i downloaded 1.12.6 besides my 1.12.7 and tried it there and it worked like it used to do. i tried to check if ANY setting or config was diff on the new version, but everything was the same

https://www.reddit.com/r/SillyTavernAI/comments/1gcljqe/comment/ltvtz8a

**Reply by AlexXi19** [reply]
Thanks 👍
